### PR TITLE
[1.6 Package Add] wine-staging-git

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

File: packages/wine-staging-git/wine-staging-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+build() {
+        cd "wine64-build"
+        ../configure \
+        --prefix='/usr' \
+        --libdir='/usr/lib' \
+        --with-x \
+        --with-gstreamer \
+        --enable-win64
+        make
```
Comment (wizard-28) [inline packages/wine-staging-git/wine-staging-git.pacscript:58]:
```suggestion
        make -j$(nproc)
```

File: packages/wine-staging-git/wine-staging-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        
+        cd "../wine32-build"
+        export PKG_CONFIG_PATH="/usr/lib32"
+        ../configure \
+        --prefix='/usr' \
+        --libdir='/usr/lib32' \
+        --with-x \
+        --with-gstreamer \
+        --with-wine64="../wine64-build"
+        make
```
Comment (wizard-28) [inline packages/wine-staging-git/wine-staging-git.pacscript:68]:
```suggestion
        make -j$(nproc)
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by D-Brox** [comment]
To be re-tested after 1.6 release due to install issues

**Comment by rain1598** [comment]
> To be re-tested after 1.6 release due to install issues

what went wrong during install?

**Comment by D-Brox** [comment]
> > To be re-tested after 1.6 release due to install issues
> 
> what went wrong during install?

`mkdir` without sudo only works with fakeroot, on 1.6

**Comment by rain1598** [comment]
I'll just chown it manually

**Comment by rain1598** [comment]
nvm chown doesn't work

**Pull request merged**